packages/shared/src/contracts/agents/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './coordinator';
 export * from './global-allow-list';
 export * from './subscription-manager';
+export * from './signingCoordinator';
```

Comment:
Let's keep consistent with our naming structure. wdyt about renaming the file and therefore changing this to:
```suggestion
export * from './signing-coordinator';
```